docs/source/simba_features.rst

```diff
@@ -21,7 +21,25 @@ The level_of_loading describes the share between an empty vehicle (0) and a full
 Vehicle Dispatch
 ----------------
 
-To allocate the rotations to vehicles, vehicles of the needed type to fulfil the rotation are used. If no suitable vehicle is available, a new vehicle is created. A vehicle is defined as "available" if it is currently not serving another rotation, has the same depot and if it had enough time after return to the depot to be charged. This "minimum standing time" at the depot is calculated using the variable min_recharge_deps_oppb or min_recharge_deps_depb from the :ref`config` together with the respective battery capacity of the vehicle and assuming the maximum available power of the depot charging stations.
+To allocate the rotations to vehicles, vehicles of the needed type to fulfil the rotation are used. If no suitable vehicle is available, a new vehicle is created. The suitability depends on the chosen value of assign_strategy. Possible options are "adaptive" and "fixed_recharge". If no value is provided "adaptive" is used.
+
+
+adaptive
+###############
+A vehicle is defined as "available" if it is currently not serving another rotation, is of the same vehicle type, has the same depot and if it had approximately enough time after return to the depot to be charged to service the next rotation. If multiple vehicles can service the same rotation, the vehicle with a lower expected soc is used. The dispatch differs between opportunity and depot rotations. A vehicle is only assigned to a depot rotation if its expected soc at time of departure is bigger than the needed soc or if the expected soc is >= desired_soc from :ref`config`. A vehicle can be assigned to an opportunity rotation if its soc is smaller than the needed soc, if the charge during the last standing time is big enough to service the rotation or >= desired_soc from :ref`config`. This can happen when the previous rotation ended with a negative soc. This design decision is made to simplify the expected workflow.
```

Comment:
- "If multiple vehicles can service the same rotation, the vehicle with a lower expected soc is used." --> what do you mean by "lower expected soc"? soc_delta of the next rotation, or of the vehicle after charging before leaving, ...? Also, I'd suggest using the official SoC capitalization.
- don't use >= in the text, use "greater than or equal" (here, "greater than" would likely be correct as well). In addition, I would use "greater than" instead of "bigger than", same for "less than" instead of "smaller than". 
- Suggestion: "A vehicle is only assigned to a depot rotation if its expected SoC at time of departure is greater than a) the SoC for this rotation or b) the desired_soc from :ref`config`. A vehicle can be assigned to an opportunity rotation if its SoC is less than the needed SoC, if the charge during the last standing time is enough to service the rotation or if the expected SoC is greater than the desired_soc from :ref`config`."

Reply:
equal or greater than is what i meant. Changed the part to your suggestion.